htdocs/lib/MapMarkers.js

```diff
@@ -0,0 +1,680 @@
+};
+
+AprsMarker.prototype.draw = function() {
+    var div = this.div;
+    var overlay = this.overlay;
+    if (!div || !overlay) return;
+
+    if (this.symbol) {
+        var tableId = this.symbol.table === '/' ? 0 : 1;
+        div.style.background = 'url(aprs-symbols/aprs-symbols-24-' + tableId + '@2x.png)';
+        div.style['background-size'] = '384px 144px';
+        div.style['background-position-x'] = -(this.symbol.index % 16) * 24 + 'px';
+        div.style['background-position-y'] = -Math.floor(this.symbol.index / 16) * 24 + 'px';
+    }
+
+    // If entity is flying at a significant altitude...
+    if (this.altitude >= 500) {
+        // r = elevation, a = rotation, <x,y> = shadow offset
+        var r = Math.round(this.altitude / 1000);
+        var a = - Math.PI * (this.course? this.course - 45 : -45) / 180;
+        var x = r * Math.cos(a);
+        var y = r * Math.sin(a);
+        div.style.filter = 'drop-shadow(' + x + 'px ' + y + 'px 0.5px rgba(0,0,0,0.5))';
+    } else {
+        div.style.filter = 'none';
+    }
+
+    if (!this.course) {
+        div.style.transform = null;
+    } else if (this.symbol && !this.isFacingEast(this.symbol)) {
+        // Airplanes and other symbols point up (to the north)
+        div.style.transform = 'rotate(' + this.course + 'deg)';
+    } else if (this.course > 180) {
+        // Vehicles and vessels point right (to the east)
+        div.style.transform = 'scalex(-1) rotate(' + (270 - this.course) + 'deg)'
+    } else {
+        // Vehicles and vessels point right (to the east)
+        div.style.transform = 'rotate(' + (this.course - 90) + 'deg)';
+    }
+
+    if (this.symbol && this.symbol.table !== '/' && this.symbol.table !== '\\') {
+        overlay.style.display = 'block';
+        overlay.style['background-position-x'] = -(this.symbol.tableindex % 16) * 24 + 'px';
+        overlay.style['background-position-y'] = -Math.floor(this.symbol.tableindex / 16) * 24 + 'px';
+    } else {
+        overlay.style.display = 'none';
+    }
+
+    if (this.opacity) {
+        div.style.opacity = this.opacity;
+    } else {
+        div.style.opacity = null;
+    }
+
+    if (this.place) this.place();
+};
+
+AprsMarker.prototype.create = function() {
+    var div = this.div = document.createElement('div');
+
+    div.style.position = 'absolute';
+    div.style.cursor = 'pointer';
+    div.style.width = '24px';
+    div.style.height = '24px';
+
+    var overlay = this.overlay = document.createElement('div');
+    overlay.style.width = '24px';
+    overlay.style.height = '24px';
+    overlay.style.background = 'url(aprs-symbols/aprs-symbols-24-2@2x.png)';
+    overlay.style['background-size'] = '384px 144px';
+    overlay.style.display = 'none';
+
+    div.appendChild(overlay);
+
+    return div;
+};
+
+AprsMarker.prototype.getAnchorOffset = function() {
+    return [0, -12];
+};
+
+AprsMarker.prototype.getInfoHTML = function(name, receiverMarker = null) {
```

Comment:
I am not sure if all of the information that this method can process is available from the current APRS parser.

In general, I would like to keep the APRS info window to stay as it is for now.

Reply:
That is probably because in my current implementation AprsMarker is also used for aircraft. If you have your own class of aircraft markers, simply move the extra code to that class, deleting it from AprsMarker. I have been planning to do it myself anyway, but did not have time for it.